src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea: () => number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape = 'triangle';
 
-}
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a === 0 || b === 0 || c === 0) {
+      throw new Error('Resulting value should be even number');
```

Comment:
**The constructors should throw new Error('your error message') if:**

`any length is <= 0`